### ISSUE TITLE
Useless validation error if go back to page where a File Upload previously filled is set

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#628]
**Description**

On a multi-pages form with a **required** `File Upload` field, I submit the page. From this next page I go back to the previous page where the `File Upload` field belongs, and I get a validation error on `Submit`.

I can see just under the field that the previously uploaded file is listed (example of injected text: `1 file uploaded.`). But if I click again on the `Next step` button, I get a validation error (`Please fill out this field.`) because the `File Upload` field is empty.

This makes no sense because this field was previously filled, and is still the correct uploaded file.

Below is an example of my reduced form setup:

```
{% set handle = 'myFormHandle' %}
{% set form = craft.formie.forms({ handle: handle }).one() %}
{% set submission = craft.formie.submissions.form(handle).user(currentUser).isIncomplete(true).one() %}
{% if submission is not empty %}
	{% do craft.formie.populateFormValues(submission.form, {}) %}
	{% do craft.formie.setCurrentSubmission(form, submission) %}
{% endif %}
{{ craft.formie.renderForm(form) }}
```

### Steps to reproduce

1. Setup a multi-pages form
2. Setup a required `File Upload` field on the first page
3. Fill in the first page of the form
4. Submit and go to the next page
5. Go back to the previous page
6. Do not edit anything, just submit this page again
7. The validation error is shown whereas the file is already uploaded

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.4.23
- Craft version: 3.7.18.2
- Multi-site: Yes

**Comment by engram-design** [comment]
Yeah, this is a great point, and somewhat of a known issue. 

A `<input type="file">` field can't be populated with a file for security reasons - browsers just simply won't allow it. Files are uploaded (for Page Reload forms) immediately when you submit each page, so once the page has been submitted the field is empty, and files you've picked have been uploaded and turned into asset elements.

I'll have a look at removing validations in this instance.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.23",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.24](https://github.com/verbb/formie/releases/tag/1.4.24)
